packages/config/src/validate/helpers.js

```diff
@@ -14,6 +15,15 @@ const isString = function (value) {
   return typeof value === 'string'
 }
 
+const isCronExpression = function (value) {
+  try {
+    CronParser.parseExpression(value)
```

Comment:
shamelessly stolen from https://github.com/quirrel-dev/quirrel/blob/main/src/shared/is-valid-cron.ts.
Should I also steal the unit tests?

Reply:
I would just add some snapshot-based tests that assert a few valid and invalid values for `schedule`.

Reply:
done! https://github.com/netlify/build/pull/3631/commits/deb05041c330c3d89fd5b85ed3654431516ce697